src/client/components/app/Transactions/components/TxError.tsx

```diff
@@ -1,62 +1,118 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import styled from 'styled-components';
 
-import { Icon, Text, WarningIcon } from '@components/common';
+import { ChevronDownIcon, ChevronUpIcon, Icon, Text, WarningIcon } from '@components/common';
 import { useAppTranslation } from '@hooks';
 
-const StyledIcon = styled(Icon)`
-  width: 4rem;
-  fill: ${({ theme }) => theme.colors.txModalColors.textContrast};
+const StyledIcon = styled(Icon)<{ errorType?: ErrorType }>`
+  width: 2rem;
+  fill: ${({ theme }) => theme.colors.txModalColors.error.color};
```

Comment:
we just do fill: currentColor, and color: inherit, so you will not need to specify it again here and only on the parent

Reply:
solved on 5c0f6e767314e1ad4f1dc71847736db858f9970f